websites/docs-smoke-test/src/pages/release-notes.js

```diff
@@ -0,0 +1,39 @@
+import { ThemeProvider } from '@commercetools-docs/gatsby-theme-docs';
```

Comment:
This is here just to have a page to use as search placeholder. No functionalities are implemented here. To avoid 404